pages/linux/csplit.md

```diff
@@ -20,6 +20,6 @@
 
 `csplit {{path/to/file}} 5 {{[-f|--prefix]}} {{prefix}}`
 
-- Split a file at a line matching a `regex`:
+- Split a file at the first line matching a `regex`:
```

Comment:
Before or after the detected line?

Reply:
I reworked the whole page to clarify where the splits occur. Please re-review and let me know if this looks good to you. (@sbrl you may want to reassess too :slightly_smiling_face:)